svc-events/events/evtsubscription.go

```diff
@@ -283,19 +283,19 @@ func (p *PluginContact) CreateEventSubscription(taskID string, sessionUserName s
 
 	result.Lock.Lock()
 	originResourceProcessedCount := len(result.Response)
-        i:=0
-        var resourceId string
+	i := 0
+	var resourceID string
 
 	for originResource, evtResponse := range result.Response {
-		OriginResource:= strings.SplitAfter(originResource,"/")
-		originResourceId:=OriginResource[len(OriginResource)-1]
-		if i==0 {
-			resourceId=originResourceId
+		OriginResource := strings.SplitAfter(originResource, "/")
```

Comment:
What is this for ?

Reply:
this is Tamanna's change, when I ran the go tools, it's formatted.